recipes/kiwipy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "kiwipy" %}
+{% set version = "0.2.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "7b7845e629718cb20f431be7b5ec7faec78bdb4d52854d76be1adef633573406" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - tornado
+    - future
+
+  run:
+    - python
+    - tornado
+    - future
+    - pika
+    - pyyaml
+
+
+test:
+  imports:
+    - kiwipy
+    - kiwipy.rmq
+  commands:
+    - python -m unittest discover
```

Comment:
Remove this please

Reply:
Ok, done.
Sorry, I wasn't aware of this - I've always tried to include runs of (short) tests in my recipes so far and this is the first time someone has taken issue with it.